### PR TITLE
Add CalibrationCompanion Plugin

### PR DESCRIPTION
**Pull request by iFrostizz**
:)

### CODE REVIEW

File: _plugins/calibrationcompanion.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: plugin
+
+id: CalibrationCompanion
```
Comment (jneilliii) [inline _plugins/calibrationcompanion.md:4]: This needs to match plugin_identifier `calibrationcompanion`.

File: _plugins/calibrationcompanion.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+homepage: https://github.com/iFrostizz/OctoPrint-CalibrationCompanion
+source: https://github.com/iFrostizz/OctoPrint-CalibrationCompanion
+archive: https://github.com/iFrostizz/OctoPrint-CalibrationCompanion/archive/master.zip
+
+
+tags:
+- calibration
+
+featuredimage: /assets/img/plugins/CalibrationCompanion/setup.png
```
Comment (jneilliii) [inline _plugins/calibrationcompanion.md:20]: adjust image folder names to be all lowercase to match id/plugin_identifier.

File: _plugins/calibrationcompanion.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# Calibration Companion
+
+This plugin was created to calibrate your 3D printer settings easily.
+It comes really handy when you want to try a new filament for example.
+Let me explain all the tabs a bit more precisely before throwing yourself in the process.
+
+## Pictures 
+
+![Calibration Companion Setup](/assets/img/plugins/CalibrationCompanion/setup.png)
```
Comment (jneilliii) [inline _plugins/calibrationcompanion.md:35]: adjust image folder names to be all lowercase to match id/plugin_identifier.

File: _plugins/calibrationcompanion.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+![Calibration Companion Retraction](/assets/img/plugins/CalibrationCompanion/retraction.png)
```
Comment (jneilliii) [inline _plugins/calibrationcompanion.md:37]: adjust image folder names to be all lowercase to match id/plugin_identifier.

**Review by jneilliii — CHANGES_REQUESTED** [review]
in addition to the changes marked above, upon first look at the plugin the update hook needs to be modified to also match the id/plugin_identifier [here](https://github.com/iFrostizz/OctoPrint-CalibrationCompanion/blob/95980fa4844c452de4338b730bbce43fd84a5dbf/octoprint_calibrationcompanion/__init__.py#L238). Should be 
```
			calibrationcompanion=dict(
```

**Comment by jneilliii** [comment]
Ok, I think I got your folder name fixed up, this can be a pain on windows. Not sure if someone knows a better trick besides not using Windows, but you have to rename the folder twice to get it all lowercase. 

Couple of things prior to full review, your plugin repo has compiled code and the pycache folder from where you just uploaded it to github. Those need to be removed @iFrostizz.

**Comment by iFrostizz** [comment]
Thank you very much, jneilliii.
I'm going to compile it myself and be more careful next time.
And I deleted the comiled code files too.

**Pull request merged**